src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -47,31 +47,64 @@
  * <pre>
  * &lt;module name=&quot;DefaultComesLast&quot;/&gt;
  * </pre>
- * <p>
- * To configure the check for skipIfLastAndSharedWithCase:
+ * <p>Example:</p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     break;
+ *   case 2:
+ *     break;
+ *   default:
+ *     break; // OK
+ * }
+ *
+ * switch (i) {
+ *   case 1:
+ *     break;
+ *   case 2:
+ *     break; // OK, no default
+ * }
+ *
+ * switch (i) {
+ *   case 1:
+ *     break;
+ *   default: // violation, 'default' before 'case'
+ *     break;
+ *   case 2:
+ *     break;
+ * }
+ *
+ * switch (i) {
+ *   case 1:
+ *   default: // violation, 'default' before 'case'
+ *     break;
+ *   case 2:
+ *     break;
+ * }
+ * </pre>
+ * <p>To configure the check to allow default label to be not last if it is shared with case:
  * </p>
  * <pre>
  * &lt;module name=&quot;DefaultComesLast&quot;&gt;
  *   &lt;property name=&quot;skipIfLastAndSharedWithCase&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- * <p>
- * Example when skipIfLastAndSharedWithCase is set to true.
- * </p>
+ * <p>Example:</p>
  * <pre>
  * switch (i) {
  *   case 1:
  *     break;
  *   case 2:
- *   default: // No violation with the new option is expected
+ *   default: // OK
  *     break;
  *   case 3:
  *     break;
  * }
+ *
  * switch (i) {
  *   case 1:
  *     break;
- *   default: // violation with the new option is expected
+ *   default: // violation, with the new option is expected
```

Comment:
`, with the new option is expected` this can be removed

Reply:
done